applications/FluidDynamicsApplication/python_scripts/navier_stokes_ale_fluid_solver.py

```diff
@@ -15,6 +15,74 @@ def CreateSolver(model, solver_settings, parallelism):
 
 
 class NavierStokesAleFluidSolver(AleFluidSolver):
-     def _CreateFluidSolver(self, solver_settings, parallelism):
+    def _CreateFluidSolver(self, solver_settings, parallelism):
         return python_solvers_wrapper_fluid.CreateSolverByParameters(
             self.model, solver_settings, parallelism)
+
+    def _SelectMeshVelocityCalculationSettings(self):
```

Comment:
In the future we could consider simplifying these settings. Probably, in 99% of the cases, we want the same time scheme for fluid and ALE. For the corner cases, it is better to extend this in separate classes rather than inserting lots of switch statements. For the 99% the ALE solver would simply get the fluid solver's time scheme (from the TimeDiscretization namespace) and we could skip duplicate parsing and then checking if the two settings are the same or not. The reason I say future is that I doubt if the fluid solvers/settings would support this now.